#NEW lteInputDate Validator hinzugefuegt

stevi84 opened the pull request:
Die required-Funktionen von lteComboTable und lteInput wurden etwas umgebaut.

Commit:
#NEW lteInputDate Validator hinzugefuegt

Die required-Funktionen von lteComboTable und lteInput wurden
leicht umgebaut.

Commit:
Merge branch 'issue_60104' of https://github.com/ExFace/AdminLteTemplate into issue_60104

kabachello commented:
Bitte verwendet für JS-Funktionsnamen statt {$this->getId()}_function_name {$this->buildJsFunctionPrefix()}_function_name. Diese Methode liefert stets Zeichenketten, die als JS-Methode valide sind, währen die Id per Definition lediglich als Variablennamen (bzw. Property-Name) zulässig sein muss.

Commit:
#REFACTOR JavaScript-Funktionsnamen umgestellt

Commit:
#FIX Rueckgabewert des ValueGetters wurde nicht entfernt

Wurde das Datum geloescht, gab der ValueGetter immer noch den
letzten Wert zurueck, da der intern gespeicherte Wert nicht
entfernt wurde.

stevi84 commented:
Ich habe die Funktionsnamen umgestellt, für die Einheitlichkeit auch gleich mit in lteInput und lteComboTable.
